### PR TITLE
Feature/tsq9ogs8 8 request list page

### PR DESCRIPTION
**Pull request by an-nguyen-jh**

### CODE REVIEW

File: src/components/DeviceRequest/DeviceRequest.js [hunk 1]
```diff
@@ -1,11 +1,207 @@
...
+  "Name",
+  "Team",
+  "Created Date",
+  "Device",
+  "Notification",
+  "",
+];
 
 function DeviceRequest() {
+  const [reload, setReload] = useState(Math.random());
```
Comment (anh-dinh-jh) [inline src/components/DeviceRequest/DeviceRequest.js]: What is the purpose of this Math.random?

File: src/utils/sort.js [hunk 2]
```diff
@@ -17,6 +17,14 @@ const sortPrototypes = {
...
     },
   },
+  createdTime: {
+    asc: (a, b) => {
+      return a.createdTime > b.createdTime ? 1 : -1;
+    },
+    desc: (a, b) => {
+      return b.createdTime > a.createdTime ? 1 : -1;
+    },
+  },
```
Comment (anh-dinh-jh) [inline src/utils/sort.js:27]: Can we combine this one with `updatedTime` sort?

Comment (an-nguyen-jh) [reply src/utils/sort.js:27]: I can not find possible ways to do this.

File: src/components/DeviceRequest/DeviceRequest.js (hunk 1 above)
Comment (an-nguyen-jh) [reply src/components/DeviceRequest/DeviceRequest.js]: It isn't necessary.

**Pull request merged**